src/adaptations/device-layer/include/Weave/DeviceLayer/ConnectivityManager.h

```diff
@@ -207,14 +207,24 @@ class ConnectivityManager
  */
 struct ConnectivityManager::ThreadPollingConfig
 {
-    uint32_t ActivePollingIntervalMS;                   /**< Interval at which the device polls its parent Thread router when
-                                                             when there are active Weave exchanges in progress. Only meaningful
-                                                             when the device is acting as a sleepy end node. */
-
-    uint32_t InactivePollingIntervalMS;                 /**< Interval at which the device polls its parent Thread router when
-                                                             when there are NO active Weave exchanges in progress. Only meaningful
-                                                             when the device is acting as a sleepy end node. */
 
+    /**
+     *  Interval at which the device polls its parent Thread router when
+     *  when there are active Weave exchanges in progress. Only meaningful
+     *  when the device is acting as a sleepy end node.
+     */
+    uint32_t ActivePollingIntervalMS;
+
+    /**
+     * Interval at which the device polls its parent Thread router when
+     * when there are NO active Weave exchanges in progress. Only meaningful
+     * when the device is acting as a sleepy end node.
+     */
+    uint32_t InactivePollingIntervalMS;
+
+    /**
+     * Clear the structure, setting all members to 0.
+     */
```

Comment:
What is the point of enforcing prefix documentation for inline definitions?  This is particularly onerous on readability.  

I see a lot of places in this PR where the requirement is suppressed.  If we don't like the way it looks, then why would we turn it on in the first place?

Reply:
This was one of the first files I changed, and I was still  evolving a style. In a few places earlier on I thought the right path forward was to switch to prefix and rely on the clang format down the road.  As the set of changes evolved, I've started using the `// clang-format off` directive more easily and I am happy to make those changes.